mlflow/R/mlflow/R/tracking-runs.R

```diff
@@ -239,6 +231,7 @@ mlflow_get_metric_history <- function(metric_key, run_id = NULL, client = NULL)
     purrr::transpose() %>%
     purrr::map(unlist) %>%
     purrr::map_at("timestamp", milliseconds_to_date) %>%
+    purrr::map_at("step", as.double) %>%
```

Comment:
why do we map step to double?

Reply:
It comes back as a string otherwise. We're already mapping `timestamp` to a double (which may be prone to overflow errors) and then to a `datetime` object.

Reply:
Makes sense, I was confused about the context.